src/DebugEngineHost.Stub/DebugEngineHost.ref.cs

```diff
@@ -485,6 +485,15 @@ public void SetProgress(int totalSteps, int currentStep, string progressText)
     public static class HostTelemetry
     {
         /// <summary>
+        /// Initialize Host Telemetry 
+        /// </summary>
+        [Conditional("LAB")]
+        public static void Initialize()
```

Comment:
I don't have a problem with the change, but if you want a slightly simpler fix, you could eliminate the API change and instead have the existing method which initializes the main thread do both tasks.